dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/security/Authorizer.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.mcp.internal.security;
+
+import io.openliberty.mcp.internal.McpTransport;
+import io.openliberty.mcp.internal.ToolMetadata;
+import io.openliberty.mcp.internal.exceptions.jsonrpc.HttpResponseException;
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ * Used for user authorization for calling and listing of tools
+ */
+public class Authorizer {
+
+    private static final boolean AUTHORIZED = true;
+    private static final boolean NOT_AUTHORIZED = false;
+
+    /**
+     *
+     * Authorizes the user without throwing an exception if authorization fails
+     *
+     * @param transport
+     * @param tmd
+     * @return true if the user has been authorized
+     */
+    public static boolean authorized(McpTransport transport, ToolMetadata tmd) {
+        return doAuthorization(transport, tmd);
+    }
```

Comment:
- we should rename this to `isAuthorized` to make it more obvious that the user needs to check the return value
- we can make the Javadoc a bit more clear

```suggestion
    /**
     * Checks whether the user is authorized to access a particular MCP Tool.
     *
     * @param transport the McpTransport
     * @param tmd the tool to check access for
     * @return {@code true} if the user is authorized to access the tool
     */
    public static boolean isAuthorized(McpTransport transport, ToolMetadata tmd) {
        return doAuthorization(transport, tmd);
    }
```